libr/core/cmd_anal.c

```diff
@@ -987,6 +988,41 @@ static void var_accesses_list(RAnalFunction *fcn, RAnalVar *var, int access_type
 	r_cons_newline ();
 }
 
+static void list_vars2(RCore *core, RAnalFunction *fcn, int type) {
```

Comment:
I think this whole function can be remove and probably you can just change a bit `list_vars`.

Reply:
yes, there's probably some duplicated code that can be cleaned up